sssom/cliques.py

```diff
@@ -87,6 +97,13 @@ def split_into_cliques(msdf: MappingSetDataFrame):
 
 
 def invert_dict(d: Dict[str, str]) -> Dict[str, str]:
+    """Invert Dictionary.
```

Comment:
What does it mean to invert a dictionary?

Reply:
What assumptions are necessary before doing this?

Reply:
assumptions:
1. key and value are literals of some kind
2. keyset contains no duplicated, valueset contains no duplicates. Great questions. The code should handle exceptions to these two assumptions.. Can we turn to issue?

Reply:
Invert a dictinary means keys become values and values become keys.

Reply:
Cool @hrshdhgd would you please incorporate this in the docs then

Reply:
6fd4230